app/routes.py

```diff
@@ -80,6 +80,10 @@ async def download_image(
     adjusted_x = round(x)
     adjusted_y = round(y)
 
+    # Apply an adjustment to the y-coordinate based on the observed difference
+    y_adjustment = 5  # Example adjustment value
```

Comment:
Let's remove this for now

Reply:
Sure, I'll remove the y-coordinate adjustment for now. I'll update the PR shortly. Thank you for the feedback!

Reply:
Sure, I'll remove the y-coordinate adjustment for now. I'll update the PR shortly. Thank you for the feedback!